src/template/wasm2c.declarations.c

```diff
@@ -48,9 +48,17 @@ static inline bool func_types_eq(const wasm_rt_func_type_t a,
 #endif
 
 #ifdef __GNUC__
-#define wasm_asm __asm__
+#define FORCE_READ_INT(var) __asm__("" ::"r"(var));
+// Clang on Mips requires "f" constraints on floats
```

Comment:
Maybe add link to upstream llvm bug here, so we can perhaps remove this one day?

Reply:
Sure, let me submit a separate PR for this

Reply:
Link: https://github.com/llvm/llvm-project/issues/64241

Reply:
https://github.com/WebAssembly/wabt/pull/2276

Reply:
@keithw @sbc100 Looks like the bug in llvm got fixed. We may have to keep an eye if we need add an llvm version check here to keep things compatible

Reply:
Version check makes sense to me yes